docs/content/blog/authelia-traefik-setup-guide/index.md

```diff
@@ -44,31 +60,33 @@ The first thing we want to do is set up the file structure. Which should look so
 We'll focus on the minimal configuration needed to work with Authelia. For advanced Traefik features and configurations, consult their documentation.
 {{< /callout >}}
 
-First, we'll set up Traefik as our reverse proxy. For detailed Traefik documentation, refer to the [official Traefik docs](https://doc.traefik.io/traefik/).
+Next, we'll set up Traefik as our reverse proxy. For detailed Traefik documentation, refer to the [official Traefik docs](https://doc.traefik.io/traefik/).
 
 #### Docker Compose
 
 
-```yaml{title="compose.yml"}
+```yaml{title="compose.yaml"}
 services:
   traefik:
     image: traefik:latest
     container_name: traefik
     restart: unless-stopped
-    depends_on:
-      - authelia
     security_opt:
       - no-new-privileges=true
+    depends_on:
+      - {{< sitevar name="host" nojs="authelia" >}}
```

Comment:
More of a question, do we want to make the proxy depend on Authelia?
Technically Authelia and the RP both have dependencies on each other.